### ISSUE TITLE
[Bug]: Can not reproduce accuracy test on bert and llama

### ISSUE THREAD

**Issue by pyz-creeper** [AntCPLab/OpenBumbleBee#15]
### Issue Type

Performance

### Modules Involved

Documentation/Tutorial/Example

### Have you reproduced the bug with SPU HEAD?

Yes

### Have you searched existing issues?

Yes

### SPU Version

0.8.0

### OS Platform and Distribution

Ubuntu20.04

### Python Version

3.10.13

### Compiler Version

gcc 11.4.0

### Current Behavior?

I tried reproduce the accuracy test on RTE and QNLI, but bert and llama result are very weird. Most of the time my model output something completely irrelevant to yes or no questions.
I think I may have chosen the wrong model to run these tests, as fine-tuned model will perform much better on these datasets. Can you specify the prompt you use to test accuracy, and which model you use? This will be very helpful for me.

### Standalone code to reproduce the issue

```Python
template=(
            "You are given two sentences. "
            "Sentence1: \"{sentence1}\" "
            "Sentence2: \"{sentence2}\" "
            "If sentence2 must be true whenever sentence1 is true, answer yes; otherwise answer no. "
            "Answer:"
        ),
# This is the python prompt template I use to test rte
# model creation
model = AutoModelForCausalLM.from_pretrained("bert-base-cased")
```

### Relevant log output

```Shell
I tried to generate as much as five tokens as possible, but most of my generate tokens are this:
['.....', '.....', '...,,', '.....', '.....', '. for for for for', ',,,, "', '.....']
['.....', '.....', '.....', '.. " ".', '.....', '.. " " "', '.....', '.....']
['.....', '.....', '.....', '.....', '. the the the the', '. " " ".', '.....', '.....']
['.....', '.....', '.....', '.....', '. the the the the', '.....', '.....', '.....']
['.....', '.....', '.. t t against', '.....', '. if if if if', '.....', '.....', '.....']
```

**Comment by fionser** [comment]
The BERT testing is here

https://github.com/AntCPLab/OpenBumbleBee/blob/ndss/examples/python/ml/flax_bert/flax_bert.py#L134

We compare the final logit with the CPU output

**Comment by pyz-creeper** [comment]
Thanks @fionser . I think I misuse the bert model to casual LM. I now reproduce the bert accuracy results successfully.
Is there any example for reproducing the llama result? I see that sequence classfication models has an adpater after the transformer backbone. Should I reproduce your experiment by fine-tuning a llama model?

**Comment by fionser** [comment]
LLaM result might be not a good example to use which is highly unstable.
To take a try. 
You can follow the instruction from 
https://github.com/secretflow/spu/tree/main/examples/python/ml/flax_llama7b
and to replace the SPU config to a 2pc config.

**Comment by pyz-creeper** [comment]
Thank you for you quick reply @fionser . I will follow the example in spu to see if it works.
